src/core_plugins/kibana/public/context/api/anchor.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+
+import { addComputedFields } from './utils/fields';
+import { createAnchorQuery } from './utils/queries';
+
+
+async function fetchAnchor(es, indexPattern, uid, sort) {
+  const indices = await indexPattern.toIndexList();
+  const response = await es.search({
+    index: indices,
+    body: addComputedFields(indexPattern, createAnchorQuery(uid, sort)),
+  });
+
+  if (_.get(response, ['hits', 'total'], 0) < 1) {
```

Comment:
Right, but I will handle that on a level closer to the UI. This function is not concerned with user interaction.